### ISSUE TITLE
Add jupyter logo to notebooks

### ISSUE THREAD

**Issue by stevejpurves** [curvenote/curvespace#24]

**Comment by stevejpurves** [comment]
@rowanc1 thoughts on this?

**Comment by stevejpurves** [comment]
@fwkoch I had a quick look at this but realized that there is not way currently to discriminate between notebooks and "articles" in curvespace. 

The information is not logged in the `mdast` tree or in `frontmatter`, should that be a field in `frontmatter` that is stored in the `.json` produced by curvenotejs?

**Comment by fwkoch** [comment]
Yeah, seems like just `jupyter: true/false` in the frontmatter makes sense. That would probably mean if someone puts that in the frontmatter of a md file it would also get added, but that seems fine (maybe even good...?).

**Comment by stevejpurves** [comment]
yeah, that would be ok do you think you'll be touching the frontmatter as part of the `curvenote init` ticket?

**Comment by stevejpurves** [comment]
my PR is probably stale on this one @fwkoch @rowanc1 ?

**Comment by stevejpurves** [comment]
update this work based on https://github.com/curvenote/curvenotejs/pull/152
